show/interfaces/__init__.py

```diff
@@ -467,8 +476,48 @@ def pm(interfacename, namespace, verbose):
     if namespace is not None:
         cmd += ['-n', str(namespace)]
 
+    cmd += ['-s', window_size, '-i', str(window_index)]
     clicommon.run_command(cmd, display_cmd=verbose)
 
+@pm.command(help="Display current PM data for the specified window size")
```

Comment:
can you please add the help text as per HLD?
60sec show cumulative pm statistics from the progressing 60sec pm window. ​
15min show cumulative pm statistics from the progressing 15min pm window.​
24hrs  show cumulative pm statistics from the progressing 24hrs pm window.

Reply:
As discussed, updated to "Show PM stats of progressing window for the given window size"